### PR TITLE
[plugin.video.attactv] v1.1.14

### PR DESCRIPTION
**Pull request by jamontes**
Hi,

Please, mark this version on Gotham repo as broken.
I'll make another PR with the add-on updated for Isengard onwards.
Many thanks in advance for your help and best regards,
jamontes.

Changelog: 1.1.14 (2019.01.25)
- Marked as broken for Gotham and Helix versions due to a missing dependency.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.attactv/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.attactv" name="attac tv" version="1.1.13" provider-name="Jose Antonio Montes (jamontes)">
+<addon id="plugin.video.attactv" name="attac tv" version="1.1.14" provider-name="Jose Antonio Montes (jamontes)">
     <requires>
         <import addon="xbmc.python" version="2.14.0"/>
         <import addon="plugin.video.youtube" version="4.4.1"/>
```
Comment (Rechi) [inline plugin.video.attactv/addon.xml]: This line has to be removed.

Comment (jamontes) [reply plugin.video.attactv/addon.xml]: Hi Rechi,
Do you mean the Youtube import dependency line, as such add-on is no longer available on Gotham and Helix repos?
```suggestion
        <import addon="plugin.video.youtube" version="4.4.1"/>
```
Please, confirm me the line I must remove from the file and so I'll do it.
Thanks a lot for your time and best regards,
jamontes.

Comment (Rechi) [reply plugin.video.attactv/addon.xml]: Yes exactly, remove this line.
```suggestion
```

Comment (jamontes) [reply plugin.video.attactv/addon.xml]: OK, no problem. I'll do it right now.
Thanks a lot for your help and best regards,
jamontes.

**Pull request merged**